### PR TITLE
Added 'Basically'

### PR DESCRIPTION
**Pull request by tvalentius**
Added 'basically'

Example : 
"Basically, gue sih setuju"

### CODE REVIEW

**Comment by ariya** [comment]
Thank you @tvalentius! Could you please add a test to the existing test suite?

File: src/unmix.js [hunk 1]
```diff
@@ -47,6 +47,7 @@
 
     var phraseDict = {
         'admission fee': 'biaya masuk',
+        'basically': 'pada dasarnya',
```
Comment (ariya) [inline src/unmix.js]: This should go to the other dictionary instead of this one.

**Comment by tvalentius** [comment]
Moved to 'wordDict'

**Comment by tvalentius** [comment]
package-lock.json file removed

**Pull request merged**